test/e2e/metrics_test.go

```diff
@@ -287,3 +289,24 @@ func (tc *testContext) ensureMonitoringIsEnabled() error {
 
 	return nil
 }
+
+// testNodeResourceUsage ensures cpu/memory usage information is retrievable from Windows nodes
+func testNodeResourceUsage(t *testing.T) {
+	_, err := NewTestContext()
+	require.NoError(t, err)
+
+	// Need at least one Windows node to run these tests, throwing error if this condition is not met
+	require.Greater(t, len(gc.nodes), 0, "test requires at least one Windows node to run")
+
+	for _, winNode := range gc.nodes {
+		t.Run(winNode.Name, func(t *testing.T) {
+			cmd := exec.Command("oc", "adm", "top", "node", winNode.Name)
+			out, err := cmd.Output()
+			require.NoError(t, err, "could not retreive resource usage statistics for node %s", winNode.Name)
+
+			// parse output for unreadable values. When metrics are not configured properly, values
+			// will be <unknown> as per https://bugzilla.redhat.com/show_bug.cgi?id=1920903
+			assert.NotContains(t, string(out), "<unknown>")
```

Comment:
Is this the only error output that is possible?

Reply:
This is the only error output I (and QE) has seen. I think 0% usage would also be worrisome as _some_ resources are always used by kubelet etc.